Client/Frontend/Browser/SearchViewController.swift

```diff
@@ -721,6 +752,35 @@ class SearchViewController: SiteTableViewController,
             twoLineCell.leftImageView.setFavicon(FaviconImageViewModel(siteURLString: site.url))
             twoLineCell.accessoryView = nil
             cell = twoLineCell
+        case .firefoxSuggestions:
+            let firefoxSuggestion = firefoxSuggestions[indexPath.row]
+            let titleText = String(format: .Search.FirefoxSuggestionTitle, firefoxSuggestion.fullKeyword, firefoxSuggestion.title)
+            twoLineCell.titleLabel.text = titleText
+            if Locale.current.languageCode == "en",
```

Comment:
Great question! 😄

> do we need to check "en-US" and/or other "en-xx" variations?

The locale check here is for bolding the full keyword in the suggestion title, not for controlling whether suggestions are visible. I noticed [search engine suggestions](https://github.com/mozilla-mobile/firefox-ios/blob/3d5e0534013893a6121d3e0a00348b3a7fe0fd3f/Client/Frontend/Browser/SearchViewController.swift#L650-L653) had the same check for bolding the remainder of the suggestion, so I copied it here. Would you like a follow-up ticket to revisit if we need to check other `en-xx` variants there?

In this case, though, I think you're totally right, and we can just drop it entirely, because...

> if the locale is important

It is! Suggest is `en-US` only for now.

Reply:
We decided not to show the full keyword for suggestions on mobile, so this entire branch can be deleted now! 🎉